RunTime Error after installing Form

### Current Behavior

I was trying to implement Form in my sveltekit app.

But the runtime keeps on telling me that I Cannot find module '@/utils' which has been used in the form folder.

I had to change '@/utils' to '$lib/utils' to make it work

Errors :

Failed to load url @/utils (resolved id: @/utils) in C:/Users/Mecanique/Desktop/lazy-student-frontend/my-app/src/lib/components/ui/form/form-native-select.svelte. Does the file exist?

Error: Cannot find module '@/utils' imported from 'C:/Users/Mecanique/Desktop/lazy-student-frontend/my-app/src/lib/components/ui/form/form-item.svelte'

    at nodeImport (file:///C:/Users/Mecanique/Desktop/lazy-student-frontend/my-app/node_modules/.pnpm/vite@4.4.9_@types+node@20.5.9/node_modules/vite/dist/node/chunks/dep-df561101.js:56010:25)
    at ssrImport (file:///C:/Users/Mecanique/Desktop/lazy-student-frontend/my-app/node_modules/.pnpm/vite@4.4.9_@types+node@20.5.9/node_modules/vite/dist/node/chunks/dep-df561101.js:55912:30)
    at eval (C:/Users/Mecanique/Desktop/lazy-student-frontend/my-app/src/lib/components/ui/form/form-item.svelte:4:37)

### Expected Behavior

It should be installed normally as other components i believe so

### Steps To Reproduce

1.Using the form component
2. pnpm run dev

### Link to Reproduction / Stackblitz

_No response_

### More Information

_No response_

--- a/apps/www/src/lib/registry/new-york/ui/form/form-description.svelte
+++ b/apps/www/src/lib/registry/new-york/ui/form/form-description.svelte
@@ -1,6 +1,6 @@
 <script lang="ts">
 	import { Form as FormPrimitive } from "formsnap";
-	import { cn } from "@/utils";
+	import { cn } from "$lib/utils";
 	import type { HTMLAttributes } from "svelte/elements";
 
 	type $$Props = HTMLAttributes<HTMLSpanElement>;

--- a/apps/www/src/lib/registry/new-york/ui/form/form-item.svelte
+++ b/apps/www/src/lib/registry/new-york/ui/form/form-item.svelte
@@ -1,5 +1,5 @@
 <script lang="ts">
-	import { cn } from "@/utils";
+	import { cn } from "$lib/utils";
 	import type { HTMLAttributes } from "svelte/elements";
 
 	type $$Props = HTMLAttributes<HTMLDivElement>;

--- a/apps/www/src/lib/registry/new-york/ui/form/form-label.svelte
+++ b/apps/www/src/lib/registry/new-york/ui/form/form-label.svelte
@@ -1,7 +1,7 @@
 <script lang="ts">
 	import type { Label as LabelPrimitive } from "bits-ui";
 	import { getFormField } from "formsnap";
-	import { cn } from "@/utils";
+	import { cn } from "$lib/utils";
 	import { Label } from "@/registry/new-york/ui/label";
 
 	type $$Props = LabelPrimitive.Props;

--- a/apps/www/src/lib/registry/new-york/ui/form/form-native-select.svelte
+++ b/apps/www/src/lib/registry/new-york/ui/form/form-native-select.svelte
@@ -1,7 +1,7 @@
 <script lang="ts">
 	import { Form as FormPrimitive } from "formsnap";
 	import { buttonVariants } from "@/registry/new-york/ui/button";
-	import { cn } from "@/utils";
+	import { cn } from "$lib/utils";
 	import { CaretSort } from "radix-icons-svelte";
 	import type { HTMLSelectAttributes } from "svelte/elements";
 

--- a/apps/www/src/lib/registry/new-york/ui/form/form-validation.svelte
+++ b/apps/www/src/lib/registry/new-york/ui/form/form-validation.svelte
@@ -1,6 +1,6 @@
 <script lang="ts">
 	import { Form as FormPrimitive } from "formsnap";
-	import { cn } from "@/utils";
+	import { cn } from "$lib/utils";
 	import type { HTMLAttributes } from "svelte/elements";
 
 	type $$Props = HTMLAttributes<HTMLParagraphElement>;

--- a/apps/www/static/registry/styles/new-york/form.json
+++ b/apps/www/static/registry/styles/new-york/form.json
@@ -26,7 +26,7 @@
     },
     {
       "name": "form-description.svelte",
-      "content": "<script lang=\"ts\">\n\timport { Form as FormPrimitive } from \"formsnap\";\n\timport { cn } from \"@/utils\";\n\timport type { HTMLAttributes } from \"svelte/elements\";\n\n\ttype $$Props = HTMLAttributes<HTMLSpanElement>;\n\tlet className: string | undefined | null = undefined;\n\texport { className as class };\n</script>\n\n<FormPrimitive.Description\n\tclass={cn(\"text-[0.8rem] text-muted-foreground\", className)}\n\t{...$$restProps}\n>\n\t<slot />\n</FormPrimitive.Description>\n"
+      "content": "<script lang=\"ts\">\n\timport { Form as FormPrimitive } from \"formsnap\";\n\timport { cn } from \"$lib/utils\";\n\timport type { HTMLAttributes } from \"svelte/elements\";\n\n\ttype $$Props = HTMLAttributes<HTMLSpanElement>;\n\tlet className: string | undefined | null = undefined;\n\texport { className as class };\n</script>\n\n<FormPrimitive.Description\n\tclass={cn(\"text-[0.8rem] text-muted-foreground\", className)}\n\t{...$$restProps}\n>\n\t<slot />\n</FormPrimitive.Description>\n"
     },
     {
       "name": "form-input.svelte",
@@ -34,15 +34,15 @@
     },
     {
       "name": "form-item.svelte",
-      "content": "<script lang=\"ts\">\n\timport { cn } from \"@/utils\";\n\timport type { HTMLAttributes } from \"svelte/elements\";\n\n\ttype $$Props = HTMLAttributes<HTMLDivElement>;\n\tlet className: string | undefined | null = undefined;\n\texport { className as class };\n</script>\n\n<div class={cn(\"space-y-2\", className)} {...$$restProps}>\n\t<slot />\n</div>\n"
+      "content": "<script lang=\"ts\">\n\timport { cn } from \"$lib/utils\";\n\timport type { HTMLAttributes } from \"svelte/elements\";\n\n\ttype $$Props = HTMLAttributes<HTMLDivElement>;\n\tlet className: string | undefined | null = undefined;\n\texport { className as class };\n</script>\n\n<div class={cn(\"space-y-2\", className)} {...$$restProps}>\n\t<slot />\n</div>\n"
     },
     {
       "name": "form-label.svelte",
-      "content": "<script lang=\"ts\">\n\timport type { Label as LabelPrimitive } from \"bits-ui\";\n\timport { getFormField } from \"formsnap\";\n\timport { cn } from \"@/utils\";\n\timport { Label } from \"@/registry/new-york/ui/label\";\n\n\ttype $$Props = LabelPrimitive.Props;\n\n\tlet className: $$Props[\"class\"] = undefined;\n\texport { className as class };\n\n\tconst { errors, ids } = getFormField();\n</script>\n\n<Label\n\tfor={ids.input}\n\tclass={cn($errors && \"text-destructive\", className)}\n\t{...$$restProps}\n>\n\t<slot />\n</Label>\n"
+      "content": "<script lang=\"ts\">\n\timport type { Label as LabelPrimitive } from \"bits-ui\";\n\timport { getFormField } from \"formsnap\";\n\timport { cn } from \"$lib/utils\";\n\timport { Label } from \"@/registry/new-york/ui/label\";\n\n\ttype $$Props = LabelPrimitive.Props;\n\n\tlet className: $$Props[\"class\"] = undefined;\n\texport { className as class };\n\n\tconst { errors, ids } = getFormField();\n</script>\n\n<Label\n\tfor={ids.input}\n\tclass={cn($errors && \"text-destructive\", className)}\n\t{...$$restProps}\n>\n\t<slot />\n</Label>\n"
     },
     {
       "name": "form-native-select.svelte",
-      "content": "<script lang=\"ts\">\n\timport { Form as FormPrimitive } from \"formsnap\";\n\timport { buttonVariants } from \"@/registry/new-york/ui/button\";\n\timport { cn } from \"@/utils\";\n\timport { CaretSort } from \"radix-icons-svelte\";\n\timport type { HTMLSelectAttributes } from \"svelte/elements\";\n\n\ttype $$Props = HTMLSelectAttributes;\n\n\tlet className: string | undefined | null = undefined;\n\texport { className as class };\n</script>\n\n<FormPrimitive.Select\n\tclass={cn(\n\t\tbuttonVariants({ variant: \"outline\" }),\n\t\t\"appearance-none bg-transparent font-normal\",\n\t\tclassName\n\t)}\n\t{...$$restProps}\n>\n\t<slot />\n</FormPrimitive.Select>\n<CaretSort class=\"absolute right-3 top-2.5 h-4 w-4 opacity-50\" />\n"
+      "content": "<script lang=\"ts\">\n\timport { Form as FormPrimitive } from \"formsnap\";\n\timport { buttonVariants } from \"@/registry/new-york/ui/button\";\n\timport { cn } from \"$lib/utils\";\n\timport { CaretSort } from \"radix-icons-svelte\";\n\timport type { HTMLSelectAttributes } from \"svelte/elements\";\n\n\ttype $$Props = HTMLSelectAttributes;\n\n\tlet className: string | undefined | null = undefined;\n\texport { className as class };\n</script>\n\n<FormPrimitive.Select\n\tclass={cn(\n\t\tbuttonVariants({ variant: \"outline\" }),\n\t\t\"appearance-none bg-transparent font-normal\",\n\t\tclassName\n\t)}\n\t{...$$restProps}\n>\n\t<slot />\n</FormPrimitive.Select>\n<CaretSort class=\"absolute right-3 top-2.5 h-4 w-4 opacity-50\" />\n"
     },
     {
       "name": "form-radio-group.svelte",
@@ -66,7 +66,7 @@
     },
     {
       "name": "form-validation.svelte",
-      "content": "<script lang=\"ts\">\n\timport { Form as FormPrimitive } from \"formsnap\";\n\timport { cn } from \"@/utils\";\n\timport type { HTMLAttributes } from \"svelte/elements\";\n\n\ttype $$Props = HTMLAttributes<HTMLParagraphElement>;\n\tlet className: string | undefined | null = undefined;\n\texport { className as class };\n</script>\n\n<FormPrimitive.Validation\n\tclass={cn(\"text-[0.8rem] font-medium text-destructive\", className)}\n\t{...$$restProps}\n/>\n"
+      "content": "<script lang=\"ts\">\n\timport { Form as FormPrimitive } from \"formsnap\";\n\timport { cn } from \"$lib/utils\";\n\timport type { HTMLAttributes } from \"svelte/elements\";\n\n\ttype $$Props = HTMLAttributes<HTMLParagraphElement>;\n\tlet className: string | undefined | null = undefined;\n\texport { className as class };\n</script>\n\n<FormPrimitive.Validation\n\tclass={cn(\"text-[0.8rem] font-medium text-destructive\", className)}\n\t{...$$restProps}\n/>\n"
     },
     {
       "name": "index.ts",

